src/Keboola/DbExtractor/Extractor/Redshift.php

```diff
@@ -92,22 +94,61 @@ public function export(array $table)
 
     protected function executeQuery($query, CsvFile $csv)
     {
-        $statement = $this->db->query($query);
+        $this->logger->info("Fetching data using DB cursor");
 
-        if ($statement === FALSE) {
-            throw new UserException("Failed to execute the provided query.");
-        }
+        $cursorName = 'exdbcursor' . intval(microtime(true));
```

Comment:
treba muzem pouzivat const pro tohle?

Reply:
kdyz se s tim nepracuje jinde nez v te metode tak mi to prijde jako zbytecne to davat do konstanty